ci/pipelines/release.yml

```diff
@@ -0,0 +1,1192 @@
+  serial: true
+  plan:
+  - aggregate:
+    - get: concourse
+      passed: [build-rc]
+      trigger: true
+    - get: version
+      passed: [build-rc]
+      trigger: true
+    - get: linux-rc
+      passed: [build-rc]
+      trigger: true
+    - get: unit-image
+      passed: [build-rc]
+      trigger: true
+  - task: terraform-smoke
+    file: concourse/ci/tasks/terraform-smoke.yml
+    params:
+      GCP_PROJECT: cf-concourse-production
+      GCP_KEY: ((concourse_smoke_gcp_key))
+      SSH_KEY: ((concourse_smoke_ssh_key))
+      DEPLOYMENT: smoke
+  - task: smoke
+    image: unit-image
+    file: concourse/ci/tasks/smoke.yml
+    input_mapping: {endpoint-info: outputs}
+
+- name: bosh-check-props
+  public: true
+  max_in_flight: 1
+  plan:
+  - aggregate:
+    - get: concourse
+      passed: [bin-smoke]
+      trigger: true
+    - get: unit-image
+      passed: [bin-smoke]
+      trigger: true
+    - get: version
+      passed: [bin-smoke]
+      trigger: true
+    - get: linux-rc
+      passed: [bin-smoke]
+      trigger: true
+    - get: concourse-release-repo
+  - task: check-props
+    file: concourse/ci/tasks/check-distribution-env.yml
+    image: unit-image
+    input_mapping: {distribution: concourse-release-repo}
+    params: {DISTRIBUTION: bosh}
+
+- name: bosh-bump
+  public: true
+  serial: true
+  plan:
+  - aggregate:
+    - get: concourse
+      passed: [bosh-check-props]
+      trigger: true
+    - get: unit-image
+      passed: [bosh-check-props]
+      trigger: true
+    - get: version
+      passed: [bosh-check-props]
+      trigger: true
+    - get: linux-rc
+      passed: [build-rc, bosh-check-props]
+      trigger: true
+    - get: windows-rc
+      passed: [build-rc]
+      trigger: true
+    - get: concourse-release-repo
+  - task: bump-concourse-blobs
+    file: concourse/ci/tasks/bump-concourse-blobs.yml
+    image: unit-image
+    params: {GCP_JSON_KEY: ((concourse_artifacts_json_key))}
+  - put: concourse-release-repo
+    params: {repository: bumped-concourse-release-repo}
+
+- name: bosh-smoke
+  public: true
+  serial: true
+  plan:
+  - aggregate:
+    # these don't trigger, to ensure that the job gets triggered by
+    # concourse-release, which is unfortunately decoupled from the resource
+    # that we 'put' to.
+    - get: concourse
+      passed: [bosh-bump]
+    - get: unit-image
+      passed: [bosh-bump]
+    - get: version
+      passed: [bosh-bump]
+    - get: concourse-release
+      trigger: true
+    - get: postgres-release
+      trigger: true
+    - get: bpm-release
+      trigger: true
+    - get: gcp-xenial-stemcell
+      trigger: true
+  - put: smoke-deployment
+    params:
+      manifest: concourse/ci/deployments/bosh-smoke.yml
+      releases:
+      - concourse-release/*.tgz
+      - postgres-release/*.tgz
+      - bpm-release/*.tgz
+      stemcells:
+      - gcp-xenial-stemcell/*.tgz
+      vars:
+        deployment_name: concourse-smoke-patch # TODO can't use release_minor
```

Comment:
would it make sense to still parametrize this so that there isn't a chance of a conflict if multiple patch pipelines run simultaneously ?